src/services/i18n.ts

```diff
@@ -149,11 +134,25 @@ export const resources = {
                     submitCreate: "Add Activity",
                     createActivity: "Create Activity",
                     removeDialogTitle: "Really remove Activity?",
-                    removeDialogMessage:
-                        "You will not be able to restore it afterwards!",
+                    removeDialogMessage: "You will not be able to restore it afterwards!",
                     from: "from",
                     to: "to",
                 },
+                tripSettings: {
+                    tripDetails: "Trip Details",
+                    submitUpdate: "Update Trip",
+                    members: "Members",
+                    leave: {
+                        button: "Leave Trip",
+                        dialogTitle: "Really leave Trip?",
+                        dialogMessage:
+                            "You will need to rejoin this Trip using an invitation link!",
+                    },
+                    removeMember: {
+                        dialogTitle: "Really remove Member?",
```

Comment:
```suggestion
                        dialogTitle: "Really remove member?",
```

Reply:
I've created #73 for resolving this issue as this is out of scope for this PR.